File: VB/DXSample/Data/ViewModel.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Linq
Imports System.Collections.Generic
Imports System.Collections.ObjectModel

Namespace DXSample
	Public Class ViewModel

		Private privateCollection As ObservableCollection(Of SimpleData)
		Public Property Collection() As ObservableCollection(Of SimpleData)
			Get
				Return privateCollection
			End Get
			Set(ByVal value As ObservableCollection(Of SimpleData))
				privateCollection = value
			End Set
		End Property

		Public Sub New()
			Collection = New ObservableCollection(Of SimpleData)()
			For i As Integer = 0 To 99
				Collection.Add(New SimpleData() With {.ID = i, .Name = "Name" & i, .Bool = i Mod 2 = 0})
			Next i
		End Sub
	End Class
End Namespace

File: CS/DXSample/Data/ViewModel.cs
using System;
using System.Linq;
using System.Collections.Generic;
using System.Collections.ObjectModel;

namespace DXSample {
    public class ViewModel {

        public ObservableCollection<SimpleData> Collection { get; set; }
        
        public ViewModel() {
            Collection = new ObservableCollection<SimpleData>();
            for (int i = 0; i < 100; i++) {
                Collection.Add(new SimpleData() { 
                    ID = i,
                    Name = "Name"+i,
                    Bool = i % 2 == 0,
                });
            }
        }
    }
}